tests/test_config.py

```diff
@@ -112,6 +112,56 @@ def test_nothing_changes_if_you_only_click_reconnect(root_window, monkeypatch):
     )
 
 
+def test_multiple_servers(alice, bob, mocker, monkeypatch, wait_until):
+    def dialog_callback(dialog):
+        [content] = dialog.winfo_children()
+        content._server_entry.delete(0, "end")
+        content._server_entry.insert(0, "localhost")
+        content._ssl_var.set(False)
+        content._nick_entry.delete(0, "end")
+        content._nick_entry.insert(0, "Alice2")
+        content._channel_entry.delete(0, "end")
+        content._channel_entry.insert(0, "#autojoin")
+        click(dialog, "Connect!")
+
+    monkeypatch.setattr("tkinter.Toplevel.wait_window", dialog_callback)
+    mocker.patch("tkinter.Menu.tk_popup")
```

Comment:
These would make more sense if moved down below just before where the monkeypatching is needed